ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -101,6 +134,14 @@ message ArchivedEvent {
   // Required
   Identifier template_id = 3;
 
+  // Map from parties to the indices of filters that matched the event
+  // for easy demultiplexing.
+  // This is only set for
+  // the transaction stream & the ACS stream but not for the
+  // transaction tree stream.
+  // Optional
+  map<string, FilterIndices> repeated uint32 matched_filter_indices = 12;
```

Comment:
This depends on the fact that for transaction/acs streams the filters are specified by party (`map<string, Filters> filters_by_party = 1;`, and `Filters` contains exactly one `InclusiveFilters`). If we ever change the structure of the transaction stream request, we'll have to adopt this.

Reply:
Do you have a suggestion for how to express it differently that would avoid that?